presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -959,6 +959,93 @@ where :math:`f(x)` is the partial density function of :math:`x`.
     The function uses the stream summary data structure proposed in the paper
     `Efficient computation of frequent and top-k elements in data streams <https://www.cse.ust.hk/~raywong/comp5331/References/EfficientComputationOfFrequentAndTop-kElementsInDataStreams.pdf>`_ by A.Metwally, D.Agrawal and A.Abbadi.
 
+Reservoir Sample Functions
+-------------------------------
+
+As opposed to using Presto's `TABLESAMPLE` operator, reservoir samples allow the
```

Comment:
```suggestion
Reservoir sample functions use a fixed sample size, as opposed to ``TABLESAMPLE``. 
```
Suggest rephrasing for conciseness and formatting, and also to put the explanation of what a reservoir sample function is first (then follow with what it is not) for easier reading comprehension. 

Also and very optional, I found TABLESAMPLE in the docs - here is a cross-link to it if you want to include it here instead. 
```
`TABLESAMPLE </sql/select.html#tablesample>`_
```

Reply:
Thanks! I linked using RST's `:ref:` label instead to prevent a broken link in the case the internal doc structure ever changes (see additional change to `sql/select.rst`)

Reply:
Nice, I like the change! I tested this in a local build just now and it works fine.